Makefile

```diff
@@ -104,13 +104,13 @@ test-singlepass-jit:
 	cargo test --release $(compiler_features) --features "test-singlepass test-jit"
 
 test-cranelift-native:
-	cargo test --release $(compiler_features) --features "test-cranelift test-native"
+	cargo test --release $(compiler_features) --features "test-cranelift test-native test-no-traps"
```

Comment:
TODO: Remove test-no-traps (based in the other comment advocating for using `test-native` directly)

Reply:
Done.